app/views/claims/mostly_teaching_eligible_subjects.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { mostly_teaching_eligible_subjects: "tslr_claim_mostly_teaching_eligible_subjects_true"}) if current_claim.errors.any? %>
+      <%= form_for current_claim, url: claim_path do |form| %>
+        <%= form.hidden_field :mostly_teaching_eligible_subjects %>
+        <%= form_group_tag current_claim do %>
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+              <h1 class="govuk-fieldset__heading">
+                <%= t("tslr.questions.mostly_teaching_eligible_subjects", subjects: subject_list(current_claim.eligible_subjects)) %>
```

Comment:
`Did you teach %{subjects} for more than 50% of your teaching time between 6 April XXXX and 5 April XXXX?` with `or` (the default) reads like an exclusive or, not an inclusive or. Is that intentional?

Reply:
I'd say so, as long as they teach one of those subjects for 50% of their time or more, then they'll be eligible

Reply:
What if they, e.g. teach Chemistry 25% and Biology 25%? Are they eligible?

Reply:
Dunno. I've popped the question into Slack.